$NetBSD$

Do not install fswatch-run-* conditionally on shell existence,
would break packaging.
TODO: install unconditionally, or add dependencies

--- Makefile.am.orig	2015-01-14 15:07:55.000000000 +0000
+++ Makefile.am
@@ -28,14 +28,9 @@ dist_doc_DATA  = README.bsd README.freeb
 dist_doc_DATA += README.linux README.osx AUTHORS COPYING LICENSE NEWS
 
 dist_bin_SCRIPTS = 
-if HAVE_ZSH
-  dist_bin_SCRIPTS += scripts/fswatch-run-zsh
-endif
-if HAVE_BASH
-  dist_bin_SCRIPTS += scripts/fswatch-run-bash
-endif
 
 install-exec-hook: fswatch-remove-links
+not:
 if HAVE_ZSH
 	$(LN_S) $(bindir)/fswatch-run-zsh $(DESTDIR)$(bindir)/fswatch-run
 else
